docs/structure/toc-generate.md

```diff
@@ -0,0 +1,42 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.6.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Generate a Table of Contents file from your book's files
+
+You can use `jupyter-book` to *generate* a table of contents file from your book
+using the filenames of your book's content. To do so, run the following command
+
+```bash
+jupyter-book toc from-site path/to/book
```

Comment:
Forgot to mention, here you need to specify the format you want, e.g. `-f jb-article` or `-f jb-book` then also you should explain the other options available (unless that is somewhere else)

Reply:
ah yeah - I added that option to the example - we show the `help` output at the bottom of the page, that work?